.travis.yml

```diff
@@ -1,18 +1,43 @@
 # System Set-Up
 language: cpp
-compiler: gcc
 
-# GitHub branch
-branches:
-  only:
-  - master
+# install dependencies
+addons:
+  apt:
+    packages:
+      - cmake
+      - libcfitsio3-dev
+  homebrew:
+    packages: &macos_packages
+      - cmake
+      - pkgconfig
+      - cfitsio
 
-sudo: false
+matrix:
+  include:
+    - os: linux
+    - os: osx
+      osx_image: xcode10.1
+      env:
+        - MATRIX_EVAL="CC=gcc-7 && CXX=g++-7"
+      addons:
+        homebrew:
+          packages:
+            - gcc@7
```

Comment:
@giordano Is there any reason why you downgraded to `gcc-7` rather than using the current Homebrew default of `gcc-8`?

Reply:
Not really, I started from a previous Travis script where we were using GCC 7.  I can upgrade to 8 if you want.

Reply:
I think that would be best as most macOS users will use the Homebrew default.

Reply:
Done!

Reply:
Well, apparently there was a reason :confused:

Reply:
Odd, it builds locally with `gcc-8` without any problems. You can revert to `gcc-7` as it seems to work on Travis.

Reply:
That might be a Travis quirk